src/Views/Pages/About/About.js

```diff
@@ -31,6 +41,53 @@ export default () => {
                 <ViewWrapper
                 />
             </div>
+        <div className={clsx(classes.header)}>
+            <h1> About Us </h1>
```

Comment:
Similar to above, use the Material UI Typography for this stuff as well. Set the prop variant='h2' and so on